components/org.wso2.carbon.identity.scim2.common/src/main/java/org/wso2/carbon/identity/scim2/common/impl/SCIMUserManager.java

```diff
@@ -719,6 +719,28 @@ private long getTotalUsers(String domainName) throws CharonException {
         return totalUsers;
     }
 
+    private long getTotalUsers(ExpressionNode node, String domainName) throws CharonException {
+
+        long totalUsers = 0;
+        AbstractUserStoreManager secondaryUserStoreManager = null;
+        if (StringUtils.isNotBlank(domainName)) {
+            secondaryUserStoreManager = (AbstractUserStoreManager) carbonUM
+                    .getSecondaryUserStoreManager(domainName);
+        }
+        try {
+            if (secondaryUserStoreManager instanceof JDBCUserStoreManager) {
```

Comment:
What will happen if the userstore manager is not an instance of  JDBCUserStoreManager? It will return the totalUsers as 0

Reply:
Yes this will return totalUsers as 0